### DISCUSSION TITLE
How do I disable version downgrade in WiX bootstrapper?

### DISCUSSION THREAD

**Discussion by sansa20** [wixtoolset/issues#7306] [WiX Development]
Here is my scenario- I have an application.msi which requires few pre-requisites to run. So, I have created a bootstrapper.exe (version 2.4) with the applcation.msi and all pre requisites bundles together. 
When I download application.msi (version 2.6 or 2.7), and then proceed to download bootstrapper.exe ( version 2.4). I want the system to show an error saying "Version downgrade is not supported". What happens instead is that on the UI, the downgrading takes place smoothly without any errors. But on the control panel and my desktop application, the bootstrapper.exe ( version 2.4) does not show up.
So basically downgrading happens only on the UI. How to I prevent this? 

Following is the code I am using right now for the .wxi file
```xml
<MajorUpgrade AllowSameVersionUpgrades="yes" DowngradeErrorMessage="Can’t downgrade to an older version." AllowDowngrades="no" />
```

**Comment by barnson** [comment]
https://wixtoolset.org/docs/schema/bal/wixstandardbootstrapperapplication/

**Comment by rseanhall** [comment]
It sounds like you're expecting things to work when a users installs the application .msi by itself then installs a bundle that contains the application .msi. That isn't really supported by any of the built-in bootstrapper applications, they are expecting the application to always be installed as a bundle.

**Reply by sansa20** [reply]
Oh I see. So in case I am trying to prevent version downgrade from bundle to bundle, is there any way I can show an error message instead of the system simply prevent downgrading by displaying a setup fail. Something like the DowngradeErrorMessage we have in case of Setup projects.
